deeds-dapp-webapp/src/main/webapp/static/i18n/messages_en.properties

```diff
@@ -287,7 +287,7 @@ deedRentingRewardDistributionForTheTenant=For the tenant
 deedRentingCreateButton=Create Offer
 deedRentingUpdateButton=Update Offer
 deedRentingUpdateNoBlockchainButton=Save Offer
-deedRentingDeleteButton=Delete Offer
+deedRentingDeleteButton=Delete
```

Comment:
This change has to go to crowdin. Only new labels can be changed from code.

Reply:
Yes and already done from crowdin @MayTekayaa 
https://crowdin.com/translate/exo-platform/38204/en-en#7051679

Reply:
done @boubaker 
Good @srenault-meeds thanks